src/main/java/bio/terra/dao/StudySummaryDao.java

```diff
@@ -0,0 +1,72 @@
+package bio.terra.dao;
```

Comment:
Summary is only ever an output from the API. It is never an input, so having a class called Summary that exposes create seems misleading. There is an implicit assumption that the summary will always contain only the data from the study table.

In the dataset case, summary will eventually contain more than the dataset table information. It may contain other data from other tables, like counts of records, histograms of ages, etc. So following this pattern of classes isn't right for dataset.

I would prefer a structure where you made a free-standing StudySummary metadata object and added dao methods to only retrieve it. That would reflect its read-only nature. Instead of making a dao for the summary.

That is my plan for the dataset implementation.

Reply:
so the dao methods to retrieve it would be on the StudyDao. and the Study metadata object would have a reference a StudyDao object (instead of inheriting) to hold those values for the full Study?

Reply:
or would the Study just have the same fields as the StudySummary and not contain one?

Reply:
What I did in dataset was to make a DatasetSummary metadata class and a retrieveDatasetSummary in the dao. It has most of the same fields as from the Dataset class. 

So what you said last: it duplicates, but doesn't contain. There is some code duplication in the SELECT that builds the DatasetSummary, but it seemed minor to me.

What you wrote works fine; it is more a question of whether it is important to make the code structure consistent between study and dataset; or not worry about it; or kick it down the road...